code/game/objects/structures/memorial.dm

```diff
@@ -1,6 +1,8 @@
 /*
  At 7:30 AM on March 8th, 2017, ArcLumin died in a fatal car crash on impact.
 Rest in peace, man. You did good work.
+A friend, contributor and member of our community,
```

Comment:
I'd add a date to this so it fits in alongside ArcLumin's memorial.
`On the 6th of November, 2018 at the age of 22`
Something like that.